### ISSUE TITLE
[Feature Request] Adding Propper to H++

### ISSUE THREAD

**Issue by Valkyrie-2154** [ficool2/HammerPlusPlus-Issue-Tracker#297]
Love the "Create Instance from Selection" button added in the context menu in H++. I was hoping that Propper could be added to H++ by default and possibly have a similar system? ie "Create Propper Model from Selection" or even just having propper included with H++.

I understand this is a fairly large request but it would make it much easier for mappers to optimize maps.

Thank you!

**Comment by Till-G** [comment]
+1 :)

**Comment by stamepicmorg** [comment]
propper is a separate standalone compiler tool and not affiliated to h++ at all.

i think this is bad idea to publish 3rtp party tools bundled with h++.

but add some support of propper if it installed correctly is nice idea.

**Comment by Valkyrie-2154** [comment]
> propper is a separate standalone compiler tool and not affiliated to h++ at all.
> 
> i think this is bad idea to publish 3rtp party tools bundled with h++.
> 
> but add some support of propper if it installed correctly is nice idea.

Yeah, I agree with that to be honest. But the ability to drop propper in and h++ having functionality built it would be amazing c:

**Comment by stamepicmorg** [comment]
you could edit hammer's sequence config and add propper. than backup config for future use. i think it' normal way.

**Comment by megakarlach** [comment]
![image](https://github.com/user-attachments/assets/668da2d2-d791-497b-a2d1-78a0543e971e)
propper++ has been added to hammer++ recently.

**Comment by stamepicmorg** [comment]
nice!
